tests/functional/watcher_controller_test.go

```diff
@@ -371,4 +372,31 @@ var _ = Describe("Watcher controller", func() {
 		})
 	})
 
+	When("Watcher CR is created without container images defined", func() {
```

Comment:
We may use any of the existing cases for this tests, i.e. When("A Watcher instance is created from minimal spec").

Reply:
Done!